UI HOVER EFFECT/code/style.css

```diff
@@ -0,0 +1,69 @@
+
+#content{
+    background-color: #f3f1fe;
+    align-items: center;
+  
```

Comment:
Please remove extra whitespaces inside and after css classes. For styling guidelines you can refer to [this](https://google.github.io/styleguide/htmlcssguide.html)